python/ray/dataframe/dataframe.py

```diff
@@ -458,8 +479,6 @@ def dropna(self, axis, how, thresh=None, subset=[], inplace=False):
             DataFrame with the dropna applied.
         """
         raise NotImplementedError("Not yet")
-        if how != 'any' and how != 'all':
```

Comment:
Why was this deleted?

Reply:
I removed it because it will never be reached because of the `NotImplementedError`